.gitignore

```diff
@@ -1,3 +1,7 @@
+# SPDX-FileCopyrightText: 2021 ladyada for Adafruit Industries
+#
+# SPDX-License-Identifier: Unlicense
+
 # SPDX-FileCopyrightText: 2017 Scott Shawcroft, written for Adafruit Industries
```

Comment:
Many of the files in this repo seem to have gotten copyright info doubled up in them. I'll come back tonight and push a change to get them back to only the single original copyright line.